airflow/hooks/dbapi_hook.py

```diff
@@ -19,8 +19,6 @@
 
 import warnings
 
-from airflow.hooks.dbapi import DbApiHook  # noqa
-
 warnings.warn(
```

Comment:
```suggestion
from airflow.providers.common.sql.hooks.sql import DbApiHook  # noqa

warnings.warn(
```

We still need to import `DbApiHook` here, no?

Reply:
Good eye. Not sure where it disappeared :)

Reply:
Should be fixed.